index.js

```diff
@@ -3,16 +3,12 @@ const config = require("./config.js");
 const cache = require("./cache.js");
 const buildServer = require("./server/build.js");
 
-const mongoose = require("mongoose");
-
 const app = buildServer();
 
+const Database = require("./utils/Database");
+
 // DB connection
-mongoose
-  .connect(config.mongoUrl, {
-    useNewUrlParser: true,
-    useUnifiedTopology: true,
-  })
+Database.connect()
```

Comment:
Database'in icersindeki console.log'lari kaldiralim. Aksi taktirde uygulama calisirken 2 sefer log yazmis oluyoruz. console.log yerine her zaman `fastify.log` kullanalim.

Reply:
tabi